cronos-play/getting-started_unreal/quick-start/wallet.md

```diff
@@ -0,0 +1,133 @@
+# Wallet
+
+Most of the wallet functionality is insecure / meant for development testing. To connect Wallets with Dapps/Dgames securely, we bring the open communications protocol for web3 into Unreal Engine. Please check [WalletConnect](walletconnect.md).
+
+The following functions are members of **DefiWalletCoreActor**. The Target should be **Defi Wallet Core Actor**.
+
+### InitializeWallet
+
+Restore wallet with mnemonics and password.
+
+<figure><img src="../../../.gitbook/assets/image (14) (1).png" alt=""><figcaption></figcaption></figure>
+
+* `Mnemonics`: mnemonics. to restore
+* `Password`: salt in mnemonics restoration
+* `Output`: generated address (index=0)
+* `Success`: whether succeed or not
+* `Output Message`: error message, "" if succeed
+
+## InitializeNewDevelopmentOnlyWallet
+
+
+
+Create a new wallet with password and wordcount.
+
+<figure><img src="../../../.gitbook/assets/image (24).png" alt=""><figcaption></figcaption></figure>
+
+* `Password`: salt in mnemonics restoration
+* `Wordcount`: mnemonics word count (12, 18, 24)
+* `Output`: generated address (index=0)
+* `Success`: whether succeed or not
+* `Output Message`: error message, "" if succeed
+
+## GetBackupMnemonicPhrase
+
+Get backup mnemonic phrase.
+
+<figure><img src="../../../.gitbook/assets/image (18) (1).png" alt=""><figcaption></figcaption></figure>
+
+* `Output`: backup mnemonics
+* `Success`: whether succeed or not
+* `Output Message`: error message, "" if succeed
+
+## DevelopmentOnlyGenerateMnemonics
```

Comment:
Since duplicate with InitializeNewDevelopmentOnlyWallet?

Reply:
DevelopmentOnlyGenerateMnemonics  only `Generate mnemonics` with password and wordcout, but no the wallet. InitializeNewDevelopmentOnlyWallet will Generate mnemonics with password and wordcout then create a wallet.

Reply:
Got it, good to know